logfire/_internal/integrations/asyncpg.py

```diff
@@ -25,7 +25,7 @@ def instrument_asyncpg(logfire_instance: Logfire, **kwargs: Unpack[AsyncPGInstru
 
     See the `Logfire.instrument_asyncpg` method for details.
     """
-    AsyncPGInstrumentor().instrument(  # type: ignore[reportUnknownMemberType]
+    AsyncPGInstrumentor().instrument(
```

Comment:
I'm so happy this got in: https://github.com/open-telemetry/opentelemetry-python-contrib/pull/3084